src/game-logic/cows/cow-selection.test.js

```diff
@@ -19,12 +20,20 @@ describe('cow selection', () => {
     const cowStub1 = generateCow({ id: cowId1 })
     const cowStub2 = generateCow({ id: cowId2 })
 
-    await farmhandStub({
+    const loadedState = saveFileStubFactory({
       cowInventory: [cowStub1, cowStub2],
-      stageFocus: stageFocusType.COW_PEN,
       purchasedCowPen: 1,
     })
 
+    await farmhandStub({
+      localforage: {
+        getItem: () => Promise.resolve(loadedState),
+        setItem: (_key, data) => Promise.resolve(data),
+      },
+    })
```

Comment:
This is the new preferred way to hydrate the game with state for testing.

Reply:
as in, do this instead of creating a farmhand context provider? and does this mean every test has to be async now?

Reply:
also can a note be added to the README about how to work with this?